Swiftfin tvOS/Components/CheckButton.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+
+struct CheckButton: View {
```

Comment:
I don't feel like this serves much purpose in the area that it's currently being used. I would also prefer we move to pickers for select-able items, especially now that we can use `Menu` or `pickerStyle(.menu)`.

Reply:
Oh nice the Menu items look a lot better! I tried to take a crack at this. Is this kind of what you were looking for:
![Simulator Screenshot - Apple TV 4K (3rd generation) - 2024-08-06 at 15 01 03](https://github.com/user-attachments/assets/2c2ad4d3-226d-46c7-8ffd-0b10772a0b0b)

Reply:
I've included this working version on my latest commit. It's rough around the edges with now and I don't like how the box has a larger button box around it but for the menu specific drop down I think that shows the basic of what I think you're suggesting.